pybossa/api/__init__.py

```diff
@@ -461,3 +463,63 @@ def task_gold(project_id=None):
     task_repo.update(task)
 
     return Response(json.dumps({'success': True}), 200, mimetype="application/json")
+
+
+@jsonpify
+@login_required
+@csrf.exempt
+@blueprint.route('/task/<task_id>/services', methods=['POST'])
```

Comment:
considering extension of this feature to support additional services in future, service name can also be passed in url
> url /task/task_id/service/my-service-name